docs/DeveloperGuide.md

```diff
@@ -157,6 +157,23 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 ### Implemented features
+#### Contact features
+**SortContact feature**
+The `sort` command (class SortContactCommand) is a SortContact feature which allows a user to sort the current person list in alphabetical order permanently. Contacts that are being added to the list later will not be sorted and added to the end of the list.
```

Comment:
i think this part is a bit confusing especially the command and features part? 
(also what is the definition of feature vs command? because I've always interpreted feature as the whole aspect (e.g. contacts, tags, tasks...), and commands to be like what you can do with those features but maybe I'm wrong?)

also, i think the command name is normally named after the command word? so this could be kind of confusing

Reply:
hmmm if like this, then we might have to change all to command only